packages/eslint-config-react/package.json

```diff
@@ -12,7 +12,7 @@
     "eslint": ">= 3"
   },
   "dependencies": {
-    "@mollie/eslint-config-base": "^0.0.1-alpha.2",
+    "@mollie/eslint-config-base": "^0.0.1-alpha.3",
```

Comment:
Nice one 👍
Leave the other versions for now. I'll version bump all of them to the correct versions upon release (after merging).